Gaijinizer patch/Scripts.txt

```diff
@@ -2957,46 +2952,46 @@ You have successfully completed this game!
 > CONTEXT: Scripts/テキスト出力/126:12
-"Subject: To 'interventor'
-sender【___】
+"Sender: Unknown
+Subject: To the 'Intervener'
 
- 《Text》==========================================
-...... Is it properly received?
- Invents a machine that sends messages directly to Dr. Kurosawa.
-I'm skeptical and I am using it ...
-If this has not arrived, it's my grand soliloquy ...
-
-You know my name, right?
- You gave me a little through Meira when I was young ...
-You made me escape from that city.
- I can not thank you.
-
- I have no intention to blame you.
-That is what Reina wanted.
-And as she says...I couldn't close the trigger.
-Well, if the decision was a little late,
- I and Reina were also included in Samael.
- Her resolution was about to be wasted.
-
-But if you say that there is no regret...it will be a lie.
- Of course, I intended to act so as not to regret myself.
-But if there was a way to save her ...
-
-One thing I would like to ask.
-If you roll around the world, you have another end to this end
-If you have the courage to see...I want you to save her again this time.
-
- I of 'this world' can not see the happy ending.
-But you just find out the possibility
-For me, it should be a big power ......
-
-Unfortunately I can not help but do not worry.
-'My girls' in my room will help you
-The power of Kizuna that you have spun up will surely not be wasted.
-
-Well then...I think that you are goodbye already ...
-Thank you to the next world 's' I'.
- =================================================="#MTLed
+
+ 《Body》==========================================
+...Is this getting through?
+I'm using this device Professor Yui invented to send a message
+directly to the 'Intervener'. Honestly I'm a little skeptical it'll
+work. If this isn't going anywhere then I'm just making a grand speech
+to myself.
+
+You know my name, right? You were the one who gave it to me through
+Meira-san. You were also the one who got me out of that city.
+I really can't thank you enough.
+
+I have no intention of blaming you for what happened, it's what Reina
+wanted. And just like she said, I wouldn't have been able to pull the
+trigger. And if you'd been just a little slower to make the decision,
+Reina and I both would've been pulled completely into Samael. Reina's
+determination was almost wasted.
+
+But I'd be lying if I said I didn't have any regrets.
+Obviously I intended to act in such a way as to not have any regrets,
+but if there'd only been a way to save her...
+
+There's one thing I'd like to ask of you.
+If you have the courage to rewind the world, and watch a different
+ending from the one we got, I want you to save her.
+
+The me of this world can't see that happy ending. But if you could
+just find that possibility for me, that'd be a huge weight off my
+shoulders.
+
+Unfortunately I can't be of any help to you, but don't worry, the
+power of the bonds you've made with the girls in 'My Room' will
+definitely come in handy.
```

Comment:
Slight changes for flow:
>_**Unfortunately I can't be of any help to you, but don't worry. The
power of the bonds you've made with the girls in 'My Room' will
definitely be of use to you.**_